probes/blocksDeleteOnBranches/impl.go

```diff
@@ -37,9 +37,26 @@ func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
 	r := raw.BranchProtectionResults
 	var findings []finding.Finding
 
+	if len(r.Branches) == 0 {
+		f, err := finding.NewWith(fs, Probe, "no branches found", nil, finding.OutcomeNotApplicable)
+		if err != nil {
+			return nil, Probe, fmt.Errorf("create finding: %w", err)
+		}
+		findings = append(findings, *f)
+		return findings, Probe, nil
+	}
+
 	for i := range r.Branches {
 		branch := &r.Branches[i]
 
+		protected := !(branch.Protected != nil && !*branch.Protected)
```

Comment:
Previously, this was done in the evaluation here:https://github.com/ossf/scorecard/blob/69bb742f12a56d223577b4984cf88b43fa56cd7d/checks/evaluation/branch_protection.go#L76-L81. Now every probe does this.

Reply:
Added in https://github.com/ossf/scorecard/pull/3759/commits/7d8ce6fd83f2445ed20b08a589eaac21d07d4487